### PR TITLE
Infinite loading fix in listing page

### PR DESCRIPTION
**Pull request by Vinod-V3**

### CODE REVIEW

File: src/app/shared/filter/filter.component.scss [hunk 1]
```diff
@@ -51,7 +51,7 @@
     background-color: var(--color-light);
     
     ion-button{
-        text-transform: none;
+        text-transform: capitalize;
```
[the remark this answers is not in the document]
Comment (Vinod-V3) [reply src/app/shared/filter/filter.component.scss:54]: By default the button label is ahown as "APPLY" but the expectation is "Apply". Hence this change

File: src/app/shared/recommendation-card/recommendation-card.component.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export class RecommendationCardComponent  implements OnInit {
   ngOnInit() {
     let evidenceListOne = this.cardData?.evidences || []
     let evidenceListTwo = this.cardData?.categories ? this.cardData.categories.flatMap((data:any) => data.evidences ?? []) : []
-    let evidenceList = [...evidenceListOne, ...evidenceListTwo]
+    let evidenceList = [...evidenceListTwo, ...evidenceListOne]
```
[the remark this answers is not in the document]
Comment (Vinod-V3) [reply src/app/shared/recommendation-card/recommendation-card.component.ts:19]: No, as per the discussion with backend the images inside categories should be the first preference. Hence this change
For the performance issue added a lazyload parameter for image tag, the datas are loaded fine, image may take 1 or 2 sec delay to load
Any how will look into this performance part later on also

File: src/app/shared/filter/filter.component.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (5Amogh) [reply src/app/shared/filter/filter.component.scss:54]: it is currently showing as Apply in filters as per expectation this capitlization will show it as APPLY

File: src/app/shared/recommendation-card/recommendation-card.component.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (5Amogh) [reply src/app/shared/recommendation-card/recommendation-card.component.ts:19]: okay

File: src/app/shared/filter/filter.component.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vinod-V3) [reply src/app/shared/filter/filter.component.scss:54]: With this change it shows as "Apply" as per the expectations

**Pull request merged**